pycaret/datasets.py

```diff
@@ -140,18 +140,23 @@ def get_data(
         save_name = filename
         data.to_csv(save_name, index=False)
 
+    display = Display(
+        verbose=True,
```

Comment:
should this be `verbose=verbose`?

Reply:
Never mind, its taken care later.

Reply:
It's actually fine, as `verbose` refers just to showing the head of the data here.